### PR TITLE
setup github actions

### PR DESCRIPTION
**Pull request by trepmal**
migrates from travis to github actions
adds phpcs

### CODE REVIEW

**Review by GaryJones — CHANGES_REQUESTED** [review]
See https://github.com/Automattic/msm-sitemap/pull/166#pullrequestreview-683774018 for feedback.

File: .phpcs.xml.dist [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	<config name="testVersion" value="5.6-"/>
+
+	<!-- Rules: WordPress Coding Standards - see
+		https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards -->
+	<!-- WordPress-Extra includes WordPress-Core -->
+	<rule ref="WordPress-Extra"/>
+	<rule ref="WordPress-Docs"/>
+	<!-- For help in understanding these custom sniff properties:
+		https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties -->
+	<config name="minimum_supported_wp_version" value="5.5"/>
```
Comment (GaryJones) [inline .phpcs.xml.dist]:
```suggestion
	<config name="minimum_supported_wp_version" value="4.5"/>
```

**Review by GaryJones — APPROVED** [review]
One suggested change, but otherwise LGTM, so approving.

File: composer.json [hunk 2]
```diff
@@ -1,42 +1,68 @@
...
+	"autoload": {
+		"classmap": [
+			"includes/"
+		]
+	},
+	"scripts": {
+		"cbf": [
+			"@php ./vendor/bin/phpcbf"
+		],
+		"coverage": [
```
Comment (GaryJones) [inline composer.json:38]: Looks like `coverage-ci` script is missing (tests failing on PHP 8.0 job).

**Review by GaryJones — APPROVED** [review]

**Comment by trepmal** [comment]
The required status check must be removed. If I'm looking at the options correct, we'll probably want to replace it with at least a few of the new checks (each WP+PHP combination from the matrix is its own)

**Comment by GaryJones** [comment]
> The required status check must be removed.

Done. Once the PR is rebased to fix the conflict, the entry for the travis-ci job requirement will disappear.

>  If I'm looking at the options correct, we'll probably want to replace it with at least a few of the new checks (each WP+PHP combination from the matrix is its own)

Agreed - these can be all of them, and can be added once the workflows are merged.

**Comment by GaryJones** [comment]
Handled with #127, #128, #129, #130.

**Pull request closed without merging**